.travis.yml

```diff
@@ -71,19 +71,25 @@ install:
   - phpenv config-add drupal.php.ini
   - phpenv rehash
 
-  # Build the Lightning code base.
+  # We need Phing to execute our build steps.
   - composer global require phing/phing
-  - phing build -Dversion=$VERSION -Dbranch=$TRAVIS_COMMIT
-
-  # Install Lightning.
-  - phing install -Ddb.database=drupal -Dversion=$VERSION
-  # Execute database updates if we're not building from HEAD.
-  - if [ $VERSION != "HEAD" ]; then phing update; fi
+  # Build the Lightning code base.
+  - composer install
+  # Install Lightning cleanly so that settings.php will be created properly.
+  - phing install -Ddb.database=drupal
+  - cd docroot
+  # If there is a database fixture for the version being tested, recreate the
+  # database from it. We need to destroy the existing database in order to
+  # prevent table collisions.
+  - FIXTURE=$TRAVIS_BUILD_DIR/tests/fixtures/$VERSION.sql.bz2
```

Comment:
Okay. I added a new target, called snapshot, which will generate the snapshot. It requires the code base to be built (i.e., git checkout followed by composer install) before it is run -- it can't do that stuff on its own because if it did a git checkout, build.xml would be overwritten with a version that hasn't got the snapshot target :)

The procedure for generating these, then, is basically this:

```
$ git checkout 8.x-1.14
$ composer install
$ phing snapshot -Dversion=1.14
```

This should create tests/fixtures/1.14.sql.bz2 from a clean install.